src/js/view/controls/components/settings/menu.js

```diff
@@ -101,7 +101,7 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
         },
         removeSubmenu(name) {
             const submenu = submenus[name];
-            if (!submenu) {
+            if (!submenu || !settingsMenuElement.hasChildNodes()) {
```

Comment:
Since we're specifically removing the submenu, it's better to use:
```
if (!submenu || !settingsMenuElement.contains(submenu.element())
```

Reply:
@egreaves agreed but `includes` doesn't work on IE

Reply:
the crash happens only when controls are false, not aware yet of a case where it crashes when controls are displayed

Reply:
We do `submenu.element().parentNode === settingsMenuElement ` in other places.

Reply:
That is the fastest and fail proof. The submenu's parent must be the menu if you expect removeChild to work. Neither function above guarantees that.